app/models/account_identity_proof.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: account_identity_proofs
+#
+#  id                :bigint(8)        not null, primary key
+#  account_id        :bigint(8)
+#  provider          :string           not null
+#  provider_username :string           not null
+#  token             :text             not null
+#  is_valid          :boolean
+#  is_live           :boolean
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+
+class AccountIdentityProof < ApplicationRecord
+
+  PROVIDER_MAP = {
+    keybase: 'Keybase'
+  }
+
+  belongs_to :account
+  validates :provider, inclusion: { in: PROVIDER_MAP.values }
+  validates :provider_username, format: { with: /\A[a-z0-9_]+\z/i }, length: { minimum: 2, maximum: 15 }
+  validates :token, length: { maximum: 66 }
```

Comment:
validate as hex: https://github.com/keybase/django-keybase-proofs/blob/master/keybase_proofs/views.py#L114

Reply:
did this